api4/config.go

```diff
@@ -279,7 +282,7 @@ func patchConfig(c *Context, w http.ResponseWriter, r *http.Request) {
 
 func makeFilterConfigByPermission(accessType filterType) func(c *Context, structField reflect.StructField) bool {
 	return func(c *Context, structField reflect.StructField) bool {
-		if structField.Type.Kind() == reflect.Struct {
+		if structField.Type.Kind() == reflect.Struct || structField.Name == FeatureFlagsElementName {
```

Comment:
Can we use an `access` tag on the `FeatureFlags` struct rather than adding this conditional logic?

Reply:
If we use the `access` tag we would either - 
- need to include all possible system console permissions
or
- only include the one necessary, but it would need updated if we used FF in `admin_definintion`, leaving us vulnerable for this bug again.
or
- introduce the concept of `access:any` 
I just thought of the last one and will explore it.

Reply:
Implemented `access:any`